Section 2/ex9/Main.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+
+int main() {
+	char input;
+	std::cout << "Hello! Please enter an alphabetical character: ";
+	std::cin >> input;
+	std::string response = (input == 97 || input == 101 || input == 105 || input == 111 || input == 117)
```

Comment:
What's the benefit of using the numbers directly rather than using the character itself? Think about legibility, if there wasn't a message to tell me what this mean, I would have no idea what these conditions are supposed to mean unless I pull up my ASCII table. Why would I need to do that if that is easily avoidable?